assets/robotest/config/pr.sh

```diff
@@ -59,18 +64,25 @@ function build_install_suite {
   local test_os="centos:7"
   local cluster_size='"flavor":"three","nodes":3,"role":"node"'
   suite+=$(cat <<EOF
- install={${cluster_size},"os":"${test_os}","storage_driver":"overlay2"}
+ install={"installer_url":"${INSTALLER_URL}",${cluster_size},"os":"${test_os}","storage_driver":"overlay2"}
 EOF
 )
   suite+=' '
-  suite+=$(build_ops_install_suite)
   echo -n $suite
 }
 
-SUITE=$(build_install_suite)
-SUITE="$SUITE $(build_resize_suite)"
-SUITE="$SUITE $(build_upgrade_suite)"
-
-echo "$SUITE" | tr ' ' '\n'
-
-export SUITE UPGRADE_VERSIONS
+if [[ ${1} == "upgradeversions" ]] ; then
```

Comment:
Maybe not in this PR but I would seriously consider moving all of this logic for generating test suite configurations into a Go program so the only shell/makefile code we have is a simple harness for building tarballs and launching robotest. Just don't want to end up in a situation with a mountain of non-trivial shell scripts to maintain.

How hard do you think it would be to do?

Reply:
I couldn't agree more.  These old shell scripts (that I've evolved considerably) are difficult to follow & debug. I don't think it would be hard.  I was actually tempted to rewrite in go it for this PR, but it felt like unnecessary scope creep.

Reply:
I've filed https://github.com/gravitational/gravity/issues/2147 tracking this work item and will triage it as I asses next workitems for https://github.com/gravitational/gravity/projects/5